pandas/core/internals/managers.py

```diff
@@ -1638,16 +1640,20 @@ def create_block_manager_from_blocks(blocks, axes: List[Index]) -> BlockManager:
         raise construction_error(tot_items, blocks[0].shape[1:], axes, e)
 
 
+# We define this here so we can override it in tests.extension.test_numpy
```

Comment:
pretty janky but i get it

Reply:
See https://github.com/pandas-dev/pandas/issues/40021